gossip3to4/addblockrequest_test.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/quorumcontrol/tupelo/gossip"
 )
 
+func init() {
+	cbornode.RegisterCborType(g3services.AddBlockRequest{})
+}
+
```

Comment:
Wouldn't this need registering in the non-test code too?

Reply:
Maybe it doesn't encounter any CBOR-encoded g3 ABRs in the non-test code? I guess it might only need it here due to the read-in from the file?